webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import qux from '/foo/bar/baz/qux.js';
+
+// import from a file named `foo.json` located in the current directory:
+import foo from './foo.json';
+
+// import from a file named `bar.js` located in the parent directory:
+import bar from '../bar.js';
+
+// import from a file named `index.js` located in a directory named `baz` that
+// is in the current directory:
+import index from './baz/index.js';
```

Comment:
Drop the `/index`. Webpack by default will look for an `index.js` file, and use that for the export